### PR TITLE
FORG-84 Intro scene

### PR DESCRIPTION
**Pull request by Marichellka**

### CODE REVIEW

File: Frogies-Project/Assets/Scripts/StorySystem/Behaviour/SimpleStoryTrigger.cs [hunk 1]
```diff
@@ -8,6 +8,7 @@ public class SimpleStoryTrigger : MonoBehaviour
     {
         [SerializeField] private StoryActor actor;
         [SerializeField] private StoryNode startNode;
+        [SerializeField] private bool autoTrigger;
```
Comment (Sorpirit) [inline Frogies-Project/Assets/Scripts/StorySystem/Behaviour/SimpleStoryTrigger.cs:11]: I dont really like this name. Maybe EnableMouseTrigger/EnablePointerTrigger

File: Frogies-Project/Assets/Scripts/WaveSystem/DayTimer.cs [hunk 2]
```diff
@@ -2,18 +2,20 @@
...
 {
     public class DayTimer : MonoBehaviour
     {
         [SerializeField] private TMP_Text timeText;
         [SerializeField] private DayNightVisuals dayNightVisuals;
+        [SerializeField] private bool isFixed;
         
         public Action OnDayEnd;
-        public bool _isDay;
+        public bool isDay;
```
Comment (Sorpirit) [inline Frogies-Project/Assets/Scripts/WaveSystem/DayTimer.cs:16]: IsDay

File: Frogies-Project/Assets/Scripts/WaveSystem/DayTimer.cs [hunk 3]
```diff
@@ -2,18 +2,20 @@
...
         
-        private float _dayDuration = 0.5f * 60;
+        private readonly float _dayDuration = 0.5f * 60;
```
Comment (Sorpirit) [inline Frogies-Project/Assets/Scripts/WaveSystem/DayTimer.cs:18]: Expose it in editor

**Review by Sorpirit — APPROVED** [review]
Over all looks good. Have left some comments

File: Frogies-Project/Assets/Scripts/StorySystem/Behaviour/SimpleStoryTrigger.cs (hunk 1 above)
Comment (Marichellka) [reply Frogies-Project/Assets/Scripts/StorySystem/Behaviour/SimpleStoryTrigger.cs:11]: this is for trigger at the beginning of the scene automatically without any action from player

File: Frogies-Project/Assets/Scripts/WaveSystem/DayTimer.cs (hunk 2 above)
Comment (Marichellka) [reply Frogies-Project/Assets/Scripts/WaveSystem/DayTimer.cs:16]: Rider, said that isDay is correct as it's not a property. But I can make it as a property

File: Frogies-Project/Assets/Scripts/WaveSystem/DayTimer.cs (hunk 3 above)
Comment (Marichellka) [reply Frogies-Project/Assets/Scripts/WaveSystem/DayTimer.cs:18]: ?

File: Frogies-Project/Assets/Scripts/StorySystem/Behaviour/SimpleStoryTrigger.cs (hunk 1 above)
Comment (Marichellka) [reply Frogies-Project/Assets/Scripts/StorySystem/Behaviour/SimpleStoryTrigger.cs:11]: I can change the name if you have any better suggestions

**Pull request merged**